bin/git-ignore-io

```diff
@@ -0,0 +1,124 @@
+#/bin/sh
+
+gitignore_io_url="https://www.gitignore.io/api/"
+
+default_path="$HOME/.gi_list"
+gi_list=`cat $default_path | tr "," "\n"`
+
+update_gi_list() {
+    curl -L -s  "${gitignore_io_url}/list" > ~/.gi_list
+}
+
+print_in_alphabetical_order() {
+    for i in {a..z};
+    do
+        echo "$gi_list" | grep "^$i" | tr "\n" " "
```

Comment:
Thanks for your advice. But, I'm wondering why sorting would be better in this case. Since I use grep to get types from a to z, it seems whether it's ordered or not doesn't matter.